hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/navBar/navBar.tsx

```diff
@@ -81,6 +81,11 @@ class NavBar extends React.Component<INavBarProps> {
             <span>Disk Usage</span>
             <Link to='/DiskUsage'/>
           </Menu.Item>
+          {/* <Menu.Item key='/HeatMap'>
```

Comment:
Remove commented code

Reply:
> Remove commented code

Done.

Reply:
@sumitagrawl  - pls check explanation to some comments and some fixed. Pls re-review.